include/net/wifi_mgmt_ext.h

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/net/wifi_mgmt.h>
+
+enum net_request_wifi_cmd_ext {
+	NET_REQUEST_WIFI_CMD_CONNECT_STORED = NET_REQUEST_WIFI_CMD_REG_DOMAIN + 100,
```

Comment:
Better to add a _MAX_ and _MAX_-1 and use that

Reply:
I can change it if https://github.com/zephyrproject-rtos/zephyr/pull/54592 gets merged in time.

Reply:
Let's do this as a future improvement after the release.